### PR TITLE
Changes in response to comments by Ariel's neuroscience colleague

### PR DESCRIPTION
**Pull request by dhuppenkothen**
Made some minor changes in response to our call yesterday and the comments forwarded by Ariel's colleague.
* removed Figure 1 and all mentions to it, also fixed references to other figure
* added a mention to reproducibility and reproducibility crisis in the introduction
* added a reference to Alex Paxton + Orianna deMasi's paper
* added a mention to coding sprints

My notes say something about adding a sentence on long-term outcomes, but I couldn't quite remember what we meant by that. There is a sentence to that effect in the results section already. @arokem @aaarendt do you remember?

### CODE REVIEW

File: paper.bib [hunk 1]
```diff
@@ -1,3 +1,63 @@
...
+pages = {1168-1169},
+year = {2015},
+doi = {10.1021/acs.jpclett.5b00285},
+    note ={PMID: 26262967},
+URL = {http://dx.doi.org/10.1021/acs.jpclett.5b00285},
+eprint = {http://dx.doi.org/10.1021/acs.jpclett.5b00285}
+}
+
+@article{pashler2012,
+author = {Harold Pashler and Eric?Jan Wagenmakers},
```
Comment (arokem) [inline paper.bib]: I believe  this question mark should be a dash (`-`)

File: paper.bib [hunk 2]
```diff
@@ -1,3 +1,63 @@
...
+title = {Editors? Introduction to the Special Section on Replicability in Psychological Science},
```
Comment (arokem) [inline paper.bib]: I believe this question mark should be an apostrophe (`'`)

**Review by arokem — COMMENTED** [review]
We should also add "... , and Jason Yeatman for comments on the manuscript" to the acknowledgements. 

Regarding long-term follow-up, I think that we can say something like: "From the very nature of the activities that we encourage in hack weeks, participants in these events produce digital records of their research online. This means that it will be fairly straightforward to evaluate the impact of these activities on their productivity (e.g., through contributions to open-source software) in the future, when sufficient time has elapsed"

**Comment by dhuppenkothen** [comment]
Thanks! I fixed the typos and added a slight modification on your sentence above to the results section.

**Comment by arokem** [comment]
👍

**Pull request merged**